src/test/java/com/iota/iri/controllers/TransactionViewModelTest.java

```diff
@@ -102,7 +111,14 @@ public void getApprovers() throws Exception {
 
     @Test
     public void fromHash() throws Exception {
+        TransactionViewModel tx = TransactionTestUtils.createBundleHead(0);
+        tx.store(tangle, snapshot);
+        Hash hash = tx.getHash();
+
+        PowerMockito.mockStatic(TransactionViewModel.class);
+        Mockito.when(TransactionViewModel.fromHash(tangle,hash)).thenReturn(tx);
 
+        assertEquals(hash, TransactionViewModel.fromHash(tangle, hash).getHash());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit assertions should include a message](https://app.codacy.com/app/GalRogozinski/iri/pullRequest?prid=4517274)